version.go

```diff
@@ -1,7 +0,0 @@
-package skopeo
-
-// Version is a version of thils build.
-const Version = "0.1.12-dev"
```

Comment:
I see, could you instead define an hash or something (even 0x01) in the signature type instead of relying on skopeo global version? are there any downsides?